test/system/tests/dictionarylockout.sh

```diff
@@ -32,6 +32,30 @@
 #;**********************************************************************;
 ###this script use for test the implementation tpm2_dictionarylockout 
 
+function yaml_get() {
```

Comment:
Probably we should at some point move all these functions to a lib.sh or something and source it from the tests. I haven't checked, but from previous review I remember quite similar functions so there's no need for the code duplication.